assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/codenvy/che.properties

```diff
@@ -69,6 +69,7 @@ schedule.core_pool_size=10
 ### that are constructed when the workspace is started. The images used to generate containers
 ### for the machines can come from DockerHub or a private Docker registry.
 machine.docker.registry=${CHE_REGISTRY_HOST}:5000
+machine.docker.registry.namespace=NULL
```

Comment:
Rename property to show its purpose
machine.docker.snapshot.registry_namespace=NULL
I would also want to rename property 1 line above but it is not backward compatible